src/pipx/commands/list_packages.py

```diff
@@ -89,12 +89,17 @@ def list_packages(
     include_injected: bool,
     json_format: bool,
     short_format: bool,
+    skip_maintenance: bool,
 ) -> ExitCode:
     """Returns pipx exit code."""
     venv_dirs: Collection[Path] = sorted(venv_container.iter_venv_dirs())
     if not venv_dirs:
         print(f"nothing has been installed with pipx {sleep}", file=sys.stderr)
 
+    if skip_maintenance:
+        shared_libs.shared_libs.skip_maintenance = True
```

Comment:
Maybe rename this to `skip_upgrade`... I don't think it's a problem to be more concise internally.